modules/goods/server/models/good.server.model.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/**
+ * Module dependencies
+ */
+var mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
+
+/**
+ * Good Schema
+ */
+var GoodSchema = new Schema({
+  name: {
+    type: String,
+    default: '',
+    trim: true,
+    required: 'Name cannot be blank'
+  },
+  description: {
+    type: String,
+    default: '',
+    trim: true
+  },
+  price: {
+    type: Number,
+    required: 'Price cannot be blank'
+  },
+  donor: {
+    firstName: {
+      type: String,
+      trim: true,
+      default: '',
+      required: 'Donor\'s first name cannot be blank'
+    },
+    lastName: {
+      type: String,
+      trim: true,
+      default: '',
+      required: 'Donor\'s last name cannot be blank'
+    },
+    email: {
+      type: String,
+      trim: true,
+      default: ''
+    },
+    receiptDate: {
+      type: Date
+    }
+  },
+  categories: [
+    {
```

Comment:
Correct me if I am wrong, this means that categories is an array of strings? like `['foo', 'bar']`